CRM/Member/DAO/Membership.php

```diff
@@ -82,6 +82,13 @@ class CRM_Member_DAO_Membership extends CRM_Core_DAO {
    */
   public $status_id;
 
+  /**
```

Comment:
DAO files are generated from the xml schema files. Check the comment at the top of the class:

```php
 /**
  * @package CRM
  * @copyright CiviCRM LLC (c) 2004-2018
  *
  * Generated from xml/schema/CRM/Member/Membership.xml
  * DO NOT EDIT.  Generated by CRM_Core_CodeGen
  * (GenCodeChecksum:643f191c50bbd0244e93745a929ba499)
  */
```

Since the GenCodeChecksum didn't change, I'm assuming you edited the file manually, so you'll probably need to generate the DAO and push it to the PR now, just in case.

Running `<civiroot>/bin/setup.sh -g` should regenerate DAO's from the updated xml schemas.

Reply:
Ran /bin/setup.sh -g and updated the DAO file as generated